src/connectors/infinite-hits/__tests__/connectInfiniteHits-test.ts

```diff
@@ -842,220 +843,146 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/infinite-hi
     });
   });
 
-  describe('routing', () => {
-    describe('getWidgetState', () => {
-      it('should give back the object unmodified if the default value is selected', () => {
-        const renderFn = jest.fn();
-        const makeWidget = connectInfiniteHits(renderFn);
-        const widget = makeWidget({ showPrevious: true });
-
-        const helper = algoliasearchHelper({} as Client, '', {});
-        helper.search = jest.fn();
-
-        widget.init!(
-          createInitOptions({
-            state: helper.state,
-            helper,
-          })
-        );
-
-        const uiStateBefore = { page: 1 };
-        const uiStateAfter = widget.getWidgetState!(uiStateBefore, {
+  describe('getWidgetState', () => {
+    test('returns the `uiState` empty without `showPrevious` option', () => {
```

Comment:
This test always pass even with `showPrevious: true` since we don't provide a page.

Reply:
The test made sense before https://github.com/algolia/instantsearch.js/pull/4040#discussion_r316740878, I didn't have the context.